### PR TITLE
Puts silicon laws on the status panel

### PR DESCRIPTION
**Pull request by imedial**
## About The Pull Request
Does as the title says
## Why It's Good For The Game
QoL feature for borgs, allows them to always see their laws without being in menus/reminded by the AI
## Changelog
:cl:
qol: puts silicon laws in the status panel
/:cl:

### CODE REVIEW

**Comment by imedial** [comment]
obligatory screenshot 
372313590-26b33963-779b-4917-84af-624aa090b6ec

**Comment by zxaber** [comment]
Kinda seems like it'll be a pain to read with long laws on widescreen.

I guess it doesn't hurt anything for it to be there as long as it's not auto-changing the status panel height.

**Comment by MrMelbert** [comment]
Maybe should go in its own tab?

**Comment by Dorsisdwarf** [comment]
A nice QOL now that checking your laws is behind pressing the PDA button, waiting for the fake loading time, then going to the law app

**Comment by imedial** [comment]
> Maybe should go in its own tab?

I feel like having it right there in the default tab is best just simply because it'll just always be up alongside all the other information that people want to be looking at frequently like round time escape time etc plus silicon laws would thematically be part of their 'status'. Personal preference on my end admittedly, but if I were to move it I feel pretty much all the silicon info should be migrated with it - charge, connected borgs/AI shells etc

**Comment by MrMelbert** [comment]
Can you post what the screen looks like when you have like 9 laws

**Comment by imedial** [comment]
> Can you post what the screen looks like when you have like 9 laws

image
Though bear in mind everyones screen is different and length of laws would matter

**Comment by imedial** [comment]
image
but even if your chat tab is really small, you can see it all with 50-50
image
problems mainly come in if you have your chat huge, but you'd have to scroll either way even if it was in its own special tab

**Comment by zxaber** [comment]
It's probably fine with the scroll. It's a quick way to just check if anything's changed, I suppose, and both borg and AI have a pop-out window for viewing laws if they want it.

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -484,3 +484,11 @@
...
 		return TRUE
 	return ..()
+
+///Places laws on the status panel for silicons
+/mob/living/silicon/get_status_tab_items()
+	. = ..()
+	var/list/law_list = list("Obey these laws:")
+	law_list += laws.get_law_list(include_zeroth = TRUE, render_html = FALSE)
+	for(var/borg_laws as anything in law_list)
+		. +=  borg_laws
```
Comment (MrMelbert) [inline code/modules/mob/living/silicon/silicon.dm:494]:
```suggestion
	for(var/borg_laws in law_list)
		. += borg_laws
```
`as anything` does nothing if you aren't typing anything

**Pull request merged**